Pansynchro.core/EventsSystem/Events/StreamEvent.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pansynchro.Core.EventsSystem.Events;
+public class StreamEvent : EventBase, IStreamEvent
+{
```

Comment:
Should this have a custom `ToString()`?

Reply:
The `ToString` method is implemented on `EventBase`, all cases are covered centrally there.